README.md

```diff
@@ -1,2 +1,146 @@
-# hello-microverse
-Hello Microverse Project
+Mylogo 
+My Company name
+
+📗 Table of Contents
+
+📖 About-This is a Hello Microverse project
+🛠 Built With HTML & CSS
+Tech Stack 
+Key Features
+🚀 Live Demo my.webpage.com
+💻 Getting Started
+Setup Download from my.webpage.com
+Prerequisites Memory Disk Processor
+Install for Mac For Win
+Usage
+Run tests
+Deployment
+👥 Authors Jose Antonio Alc1
+🔭 Future Features
+🤝 Contributing 
+⭐️ Show your support at donations.webpage.com
+🙏 Acknowledgements
+❓ FAQ (OPTIONAL)
+📝 License version 1.0
+📖 [your_project_name] Hello Microverse
+
+Describe your project in 1 or 2 sentences.
+[your_project__name] is a...
+Home page showing a html welcome page with CSS styles
+🛠 Built With HTML & CSS
+
+Tech Stack Cliente
+
+Describe the tech stack and include only the relevant sections that apply to your project.
+Client *
+Server
+Database
+Key Features
+
+Describe between 1-3 key features of the application.
+[key_feature_1]
+[key_feature_2]
+[key_feature_3]
+(back to top)
+
+🚀 Live Demo
+
+Add a link to your deployed project.
+Live Demo Link
+(back to top)
+
+💻 Getting Started
+
+Describe how a new developer could make use of your project.
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+
+In order to run this project you need:
+
+Setup
+
+Clone this repository to your desired folder:
+
+Install
+
+Install this project with:
+
+Usage
+
+To run the project, execute the following command:
+
+Run tests
+
+To run tests, run the following command:
+
+Deployment
+
+You can deploy this project using:
+
+(back to top)
+
+👥 Authors
+
+Mention all of the collaborators of this project.
+👤 Author1 Jose Antonio Alc1
+
+GitHub: @githubhandle
+Twitter: @twitterhandle
+LinkedIn: LinkedIn
+👤 Author2
+
+GitHub: @githubhandle
+Twitter: @twitterhandle
+LinkedIn: LinkedIn
+(back to top)
+
+🔭 Future Features
+
+Describe 1 - 3 features you will add to the project.
+ [new_feature_1]
+ [new_feature_2]
+ [new_feature_3]
+(back to top)
+
+🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+(back to top)
+
+⭐️ Show your support
+
+Write a message to encourage readers to support your project
+If you like this project...
+
+(back to top)
+
+🙏 Acknowledgments 
+Thanks to everyone at microverse to make this possible.
+
+Give credit to everyone who inspired your codebase.
+I would like to thank...
+
+(back to top)
+
+❓ FAQ (OPTIONAL)
+
+Add at least 2 questions new developers would ask when they decide to use your project.
+[Question_1]
+
+[Answer_1]
+[Question_2]
+
+[Answer_2]
+(back to top)
+
+📝 License
+
+This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
+
+(back to top)
```

Comment:
- [ ] Please remove `_Note_` from the` LICENCE `section of the `README`. _`Note`_ is there in the template to guide you on how you to add a LICENCE to your README. Once you have successfully added the LICENCE you can go ahead and remove it